index.html

```diff
@@ -1,226 +1,375 @@
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-199543711-1"></script>
+	<script>
+	   window.dataLayer = window.dataLayer || [];
+   
+	   function gtag() {
+		 dataLayer.push(arguments);
+	   }
+	   gtag('js', new Date());
+   
+	   gtag('config', 'UA-199543711-1');
+	</script>
   </head>
   <body>
-    <div class="hero">
-		<header class="header bg-white">
-			<nav class="navbar flex">
-				  <a href="#" class="navbar__link link fz-18 fw-700 lh-20">JUP</a>
-				<div class="navbar__menu">
-				  <ul class="navbar__menu__ul ul">
-					<li class="navbar__menu__li">
-					  <a href="#" class="navbar__menu__link link">Portfolio</a>
-					</li>
-					<li class="navbar__menu__li">
-					  <a href="#" class="navbar__menu__link link">About</a>
-					</li>
-					<li class="navbar__menu__li">
-					  <a href="#" class="navbar__menu__link link">Contact</a>
-					</li>
-				  </ul>
-				</div>
-				<img class="navbar__hamb" src="Image/hamburguer.png" alt="hamburguer icon">
-		  </nav>
-	  </header>
-
-      <section class="head flex">
-        <h1 class="head__title fz-40 fw-700 lh-52">I'm Juhar glad to see you!</h1>
-        <p class="head__dsc fz-16 fw-400 lh-24">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-		<a href="#" class="head__link link fz-16 fw-500 lh-24 clr-p-400">My Projects</a>
-		<div class="head__icons">
-			<ul class="head__icons__ul ul flex">
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/twitter.png" alt="twitter icon"></a></li>
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/Linkedin.svg" alt="linkedin icon"></a></li>
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/M.png" alt="M icon"></a></li>
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/github.svg" alt="github icon"></a></li>
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/hand.svg" alt="hand icon"></a></li>
+	<div class="hero">
+	  <header class="header bg-white">
+		<nav class="navbar flex">
+		  <a href="#" class="navbar__link link fz-20 fw-700 lh-20">JUP</a>
+		  <div class="navbar__menu">
+			<ul class="navbar__menu__ul ul flex">
+			  <li class="navbar__menu__li">
+				<a href="#works" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+			  </li>
+			  <li class="navbar__menu__li">
+				<a href="#about" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">About</a>
+			  </li>
+			  <li class="navbar__menu__li">
+				<a href="#contact" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Contact</a>
+			  </li>
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please fix the below issue and make sure your design matches the Figma design.
  - Please make sure the mobile menu options are starting on the same line vertically.
  - Please apply the blur effect in the background. `hint: backdrop-filter: blur(Npx);` N is referred as number for pixels.

### Your design
![image](https://user-images.githubusercontent.com/83803414/188828559-20ea3290-f34b-496a-9544-d2ed905f425d.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/188828619-47e30282-2068-468d-acb2-e45e11aa8bcd.png)